src/mlpack/methods/ann/layer/dropout_layer.hpp

```diff
@@ -231,7 +250,7 @@ template <
 >
 using DropoutLayer2D = DropoutLayer<InputDataType, OutputDataType>;
 
-} // namespace ann
-} // namespace mlpack
+}; // namespace ann
+}; // namespace mlpack
```

Comment:
This is pedantic (literally), but if you compile with `-pedantic` the extra semicolon produces a warning.  So I've been trying to avoid doing that. :)